src/bindings/python/CMakeLists.txt

```diff
@@ -115,6 +115,8 @@ set(INIT_FILES_RUNTIME "${OpenVINOPython_SOURCE_DIR}/src/openvino/__init__.py"
 ov_check_init_files_alignment("${INIT_FILES_RUNTIME}")
 
 ov_option(ENABLE_PYTHON "Enables OpenVINO Python API build" ${ENABLE_PYTHON_DEFAULT})
+# we should not cache the option to avoid situations when ENABLE_PYTHON is ON but ENABLE_PYTHON_DEFAULT is OFF
+unset(ENABLE_PYTHON CACHE)
```

Comment:
You can run conda build from your fork, so you don't need to wait for this to be merged. Additionally, that would verify, that your fix is enough,

Reply:
You just can even introduce patch-file, without merging anything.

My suggestion to go with patch-file and official repo